en/user_guide/media.rst

```diff
@@ -0,0 +1,57 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel
+
+Folder
+------
+
+All files are stocked in folder. It's possible to move in the folder tree to see files by click on the left panel.
+
+Create
+~~~~~~
+
+To creating folder, you can click on "new folder" button, in the left panel, where you want it.
```

Comment:
Maybe something like "Creating a folder is done by clicking on [...]"
"Where you want it" seems not appropriate for a doc